packages/client/src/components/GrantsTable.vue

```diff
@@ -138,6 +138,18 @@ export default {
       return this.grantsPagination ? this.grantsPagination.lastPage : 0;
     },
     formattedGrants() {
+      function setCellVariants(grant) {
+        const agingThreshold = 1000 * 60 * 60 * 24 * (process.env.VUE_APP_AGING_THRESHOLD_DAYS || 21);
+        const result = {};
+        const diff = new Date(grant.close_date) - new Date();
+        if (diff <= 0) {
```

Comment:
Probably a question for the partners, but I wonder if it would be more helpful to gray out grants that are already closed, and highlight in red at a different threshold, like 7 days or something.

Reply:
Good point. I'm not certain if closed grants will actually show in the application anyway. There are closed grants at grants.gov, and the cron job does not seem to load those. But I'm not certain if there is a mechanism to purge loaded grants after they have closed. @Rapol ?